packages-openshift.yaml

```diff
@@ -31,7 +31,10 @@ postprocess:
     # repos will be attempted to be fetched by rpm-ostree when doing node-local
     # kernel overrides today for e.g. kernel-rt.
     for x in $(find /etc/yum.repos.d/ -name '*.repo'); do
-      sed -i -e s,enabled=1,enabled=0, $x
+      # ignore repo files that are mountpoints since they're likely secrets
+      if ! findmnt "$x" &>/dev/null; then
+        sed -i -e s,enabled=1,enabled=0, $x
+      fi
```

Comment:
findmt seems a bit ugly here, but I guess that's the best option?

only other thing I can think of is checking to see if a file is read-only before attempting to disable the repo

Reply:
`/usr/bin/mountpoint` is also a bit more direct of an API

Reply:
Ahh yes, forgot about `mountpoint`. Will switch to that in a future patch.

Reply:
> Ahh yes, forgot about `mountpoint`. Will switch to that in a future patch.

Part of https://github.com/openshift/os/pull/1735.